apps/vessel-history/features/profile/components/ForcedLabor.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo } from 'react'
+import cx from 'classnames'
+import { Trans, useTranslation } from 'react-i18next'
+import { RiskLevel, RiskOutput, VesselWithHistory } from 'types'
+import { VesselFieldLabel } from 'types/vessel'
+import DataAndTerminology from 'features/data-and-terminology/DataAndTerminology'
+import styles from './Info.module.css'
+
+interface InfoProps {
+  vessel: VesselWithHistory | null
+}
+
+const ForcedLabor: React.FC<InfoProps> = (props): React.ReactElement => {
+  const vessel = props.vessel
+  const { t } = useTranslation()
+
+  const riskModel: RiskOutput[] = useMemo(() => {
+    if (!vessel.forcedLabour) {
+      return []
+    }
+    const yearsLength = new Date().getFullYear() - 2011
```

Comment:
I'd avoid using magic numbers like 2011 or 2012, thgere are already two constants defined in the [data/.config.ts](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/vessel-history/data/config.ts#L31-L32) that you can use here: `FIRST_YEAR_OF_DATA` and `CURRENT_YEAR`